src/bin/vip-import-sql-status.js

```diff
@@ -47,7 +48,9 @@ command( {
 
 	await track( 'import_sql_check_status_command_execute' );
 
-	console.log( `Checking the sql import status for env ID: ${ env.id }, app ID: ${ env.appId }` );
+	const progressTracker = new ProgressTracker( [] );
+	progressTracker.prefix = `=============================================================
+Checking the sql import status for env ID: ${ env.id }, app ID: ${ env.appId }:\n`;
```

Comment:
I think we should format and print these details to match the `import sql` command so it looks cleaner.

We can add that in a separate PR and just remove these lines here.

```suggestion
```